app/services/responses.js

```diff
@@ -1,30 +1,20 @@
-const { format, parseISO } = require('date-fns')
 const { getResponses } = require('../api/responses')
 
-const formatResponse = (response) => {
-  const date = parseISO(response.timestamp)
-  const generatedOn = format(date, 'dd/MM/yyyy HH:mm')
+const getDocumentResponsesHistoryData = async (id) => {
+    try {
+        const responses = await getResponses(
+          id
+        )
 
-  return {
-    response: response.response,
-    documentId: response.document_id,
-    llm: response.llm,
-    userPrompt: response.userPrompt,
-    citations: [],
-    generatedOn
-  }
-}
-
-const getLatestResponse = async (documentId) => {
```

Comment:
Is there any reason this had to be removed? The `document-response` route depends on it.

Reply:
fixed